### PR TITLE
Add fix for ListViewLeftSide.tpl when array doesn't exist

### PR DESCRIPTION
**Pull request by mlorencik**
Add fix for ListViewLeftSide.tpl when SHOW_TIMELINE_IN_LISTVIEW array doesn't exist

### CODE REVIEW

File: layouts/basic/modules/Documents/ListViewLeftSide.tpl [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		</div>
 	{/if}
 	<div>
-		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW')) && $MODULE_MODEL->isPermitted('TimeLineList')}
+		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```
Comment (rskrzypczak) [inline layouts/basic/modules/Documents/ListViewLeftSide.tpl]:
```suggestion
		{if in_array($MODULE_NAME, \App\Config::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```

File: layouts/basic/modules/Vtiger/ListViewLeftSide.tpl [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		</div>
 	{/if}
 	<div>
-		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW')) && $MODULE_MODEL->isPermitted('TimeLineList')}
+		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```
Comment (rskrzypczak) [inline layouts/basic/modules/Vtiger/ListViewLeftSide.tpl]:
```suggestion
		{if in_array($MODULE_NAME, \App\Config::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```

File: modules/OpenStreetMap/models/Module.php [hunk 3]
```diff
@@ -18,7 +18,7 @@ class OpenStreetMap_Module_Model extends Vtiger_Module_Model
 	 */
 	public function isAllowModules($moduleName)
 	{
-		return in_array($moduleName, AppConfig::module($this->getName(), 'ALLOW_MODULES'));
+		return in_array($moduleName, AppConfig::module($this->getName(), 'ALLOW_MODULES', []));
```
Comment (rskrzypczak) [inline modules/OpenStreetMap/models/Module.php]:
```suggestion
		return in_array($moduleName, \App\Config::module($this->getName(), 'ALLOW_MODULES', []));
```

File: layouts/basic/modules/Documents/ListViewLeftSide.tpl [hunk 4]
```diff
@@ -30,10 +30,10 @@
 		</div>
 	{/if}
 	<div>
-		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW')) && $MODULE_MODEL->isPermitted('TimeLineList')}
+		{if in_array($MODULE_NAME, \AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Documents/ListViewLeftSide.tpl]:
```suggestion
		{if in_array($MODULE_NAME, 
\App\Config::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```

File: layouts/basic/modules/Documents/ListViewLeftSide.tpl [hunk 5]
```diff
@@ -30,10 +30,10 @@
...
 			<a data-url="{$LISTVIEW_ENTRY->getTimeLineUrl()}" class="c-badge__icon fa-fw timeLineIconList d-none"></a>
 		{/if}
-		{if AppConfig::module('ModTracker', 'UNREVIEWED_COUNT') && $MODULE_MODEL->isPermitted('ReviewingUpdates') && $MODULE_MODEL->isTrackingEnabled() && $LISTVIEW_ENTRY->isViewable()}
+		{if \AppConfig::module('ModTracker', 'UNREVIEWED_COUNT') && $MODULE_MODEL->isPermitted('ReviewingUpdates') && $MODULE_MODEL->isTrackingEnabled() && $LISTVIEW_ENTRY->isViewable()}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Documents/ListViewLeftSide.tpl]:
```suggestion
		{if \AppConfig
\App\Config::module('ModTracker', 'UNREVIEWED_COUNT') && $MODULE_MODEL->isPermitted('ReviewingUpdates') && $MODULE_MODEL->isTrackingEnabled() && $LISTVIEW_ENTRY->isViewable()}
```

File: layouts/basic/modules/Vtiger/ListViewLeftSide.tpl [hunk 6]
```diff
@@ -26,10 +26,10 @@
 		</div>
 	{/if}
 	<div>
-		{if in_array($MODULE, AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW')) && $MODULE_MODEL->isPermitted('TimeLineList')}
+		{if in_array($MODULE_NAME, \AppConfig::module('ModTracker', 'SHOW_TIMELINE_IN_LISTVIEW', [])) && $MODULE_MODEL->isPermitted('TimeLineList')}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/ListViewLeftSide.tpl]: ?

File: layouts/basic/modules/Vtiger/ListViewLeftSide.tpl [hunk 7]
```diff
@@ -26,10 +26,10 @@
...
 			<a  data-url="{$LISTVIEW_ENTRY->getTimeLineUrl()}" class="c-badge__icon fa-fw timeLineIconList d-none"></a>
 		{/if}
-		{if AppConfig::module('ModTracker', 'UNREVIEWED_COUNT') && $MODULE_MODEL->isPermitted('ReviewingUpdates') && $MODULE_MODEL->isTrackingEnabled() && $LISTVIEW_ENTRY->isViewable()}
+		{if \AppConfig::module('ModTracker', 'UNREVIEWED_COUNT') && $MODULE_MODEL->isPermitted('ReviewingUpdates') && $MODULE_MODEL->isTrackingEnabled() && $LISTVIEW_ENTRY->isViewable()}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/ListViewLeftSide.tpl]: ?

File: modules/OpenStreetMap/models/Module.php [hunk 8]
```diff
@@ -18,7 +18,7 @@ class OpenStreetMap_Module_Model extends Vtiger_Module_Model
 	 */
 	public function isAllowModules($moduleName)
 	{
-		return in_array($moduleName, AppConfig::module($this->getName(), 'ALLOW_MODULES'));
+		return in_array($moduleName, \AppConfig::module($this->getName(), 'ALLOW_MODULES', []));
```
Comment (mariuszkrzaczkowski) [inline modules/OpenStreetMap/models/Module.php]: ?

File: modules/OpenStreetMap/models/Module.php [hunk 9]
```diff
@@ -28,7 +28,7 @@ public function isAllowModules($moduleName)
 	 */
 	public function getAllowedModules()
 	{
-		$allAllowedModules = AppConfig::module($this->getName(), 'ALLOW_MODULES');
+		$allAllowedModules = \AppConfig::module($this->getName(), 'ALLOW_MODULES');
```
Comment (mariuszkrzaczkowski) [inline modules/OpenStreetMap/models/Module.php]: ?

**Pull request merged**